web/app.php

```diff
@@ -10,16 +11,13 @@
     $debug = true;
 }
 
-/**
- * @var Composer\Autoload\ClassLoader
- */
-$loader = require __DIR__ . '/../app/autoload.php';
-
-if ($env != 'dev') {
+require __DIR__ . '/../vendor/autoload.php';
+if (PHP_VERSION_ID < 70000) {
```

Comment:
Voorlopig wel denk ik, aangezien minimum versie op 5.5 staat in onze composer momenteel